tests/models/stock_test.py

```diff
@@ -475,3 +475,97 @@ def test_should_not_include_cancelled_booking(self, app):
         # Then
         assert stock.bookingsQuantity == 1
 
+
+class IsEventExpiredTest:
+    def test_isEventExpired_is_false_when_stock_is_not_an_event(self):
```

Comment:
```suggestion
    def test_return_false_when_stock_is_not_an_event(self):
```
Peut on ne pas avoir de technique dans l’intitulé des tests ?

Reply:
```suggestion
    def test_is_not_expired_when_stock_is_not_an_event(self):
```